ur_robot_driver/config/ur10_controllers.yaml

```diff
@@ -60,3 +60,58 @@ pos_traj_controller:
    stop_trajectory_duration: 0.5
    state_publish_rate: 125
    action_monitor_rate: 10
+
+vel_based_pos_traj_controller:
+   type: velocity_controllers/JointTrajectoryController
+   joints:
```

Comment:
Could this (and other files) not use the `*robot_joints` 'trick'?

Reply:
This is what I meant by "I will start cleaning up this branch (And the controller files)" above :-)